lib/sqs.js

```diff
@@ -58,12 +133,19 @@ module.exports = ({ account, queuePrefix = '', queueSuffix = '', defaultVisibili
     }),
 
     /**
-     * Publishes a message to a queue
-     * @param queueName Name of queue
-     * @param payload body of message
-     * @param attrs attributes attached to message
+     * Publishes an extended message to a queue, meaning: if the message is
+     * too large to fit in sqs, it will write a sqs with a pointer to an s3
+     * bucket
+     *
+     * @param queueName  Name of queue
+     * @param payload    body of message
+     * @param attrs      attributes attached to message
+     * @param s3Bucket   bucket name for message storage, if it exceeds limit
+     * @param prefix     bucket path prefix (no slashes)
+     * @param shards     number of bucket shard paths
+     * @param expiration hours to retain the message before expiring
      */
-    send: ({ queueName, payload, attrs }) => {
+    extendedSend: ({ queueName, payload, attrs, s3Bucket = 'sp-sqs-extended', prefix = 'g', shards = 50, expiration = 48 }) => {
```

Comment:
maybe getting ahead of ourselves, but it might be nice to have the promise chain return some indication of whether or not it was able to fit the message in the queue?

Just thinking for observability purposes we'll probably want to be able to track that sort of thing